extensions/functions_aggregate_generic.yaml

```diff
@@ -35,3 +35,41 @@ aggregate_functions:
             value: any
         nullability: DECLARED_OUTPUT
         return: any?
+  - name: "first"
```

Comment:
Should we call out to first_value with the difference to make it easier for folks to choose one or the other?

Reply:
Just to make sure I understand, what do you see as the difference? My hope would be to replace the window versions with these completely, given that aggregate functions are also valid window functions, and the engines I looked at don't seem to make a difference between these - but maybe I missed something

Reply:
The only difference I see is what context they can be called in.